man/systemd.unit.xml

```diff
@@ -228,7 +228,9 @@
     Configurations in <filename><replaceable>type</replaceable>.d/</filename> have the lowest precedence
     compared to settings in the name specific override directories. So the contents of
     <filename>foo-.service.d/10-override.conf</filename> would override
-    <filename>service.d/10-override.conf</filename>.</para>
+    <filename>service.d/10-override.conf</filename>. In order to disable a top-level drop-in for a specific unit
+    it suffices to create a symlink to <filename>/dev/null</filename> from the unit drop-in directory. This symlink
+    should have the same name as the top-level drop-in file.</para>
```

Comment:
i think we should more explicitly mention the issue people ran into: something akin to this:

"Note that per-unit-type drop-ins generally apply to *all* units of the same type. This means that if you use such a drop-in to instantiate for each unit a secondary helper unit of the same type (e.g. consider a drop-in for service units with an `OnFailure=myfailurehandler@%N.service` line) the drop-in will also apply to that secondary helper unit, and then create another helper unit for that, and so on, until the unit name grows beyond the unit name limit. To break this recursion consider masking the per-unit-type drop-in in a drop-in specific to the secondary helper unit, by creating a /dev/null symlink with the same name as the per-unit-type drop-in."

And then maybe even give a brief exmaple?

Reply:
Thanks, I've made an attempt at this. PTAL!

Reply:
Looks good, also checked the rendered version